### ISSUE TITLE
documentation generation

### ISSUE THREAD

**Issue by zargot** [Big-Life-Lab/PHES-ODM-Validation#15]
Use Sphinx to generate documentation via github actions.
- Include quarto docs, converted to markdown.
- ~~Add a project/config file for Quarto, so that we can render all quarto-parts with a single Quarto command.~~
- Make sure that the readme link anchor works (after #75).

**Comment by zargot** [comment]
It seems like [Doxygen](https://doxygen.nl/manual/docblocks.html#pythonblocks) has support for python, and that the github-actions marketplace has good support for it. Should I go ahead with that or did you have something else in mind, @DougManuel @yulric ?

**Comment by zargot** [comment]
Something tells me [Sphinx](https://www.sphinx-doc.org/) is a more popular choice. It seems to have good support as well.

**Comment by DougManuel** [comment]
I only have a bit of experience with Doxygen - which I like. I'll defer to others.

**Comment by yulric** [comment]
Personally, I would go with Sphinx, it looks nicer haha. We'll need to decide what we want to include in the doc site for this package. Currently, I think we need the API docs and the how to tutorial. Ideally, whatever doc builder we use has support for including different documentation parts into one website.

**Comment by zargot** [comment]
I'll give Sphinx a try

**Comment by zargot** [comment]
I've updated the issue description to reflect that we're gonna write everything in Quarto and abandon Sphinx. The reason for this is to reduce complexity and make it better in general, as a mix of two doc-generation tools became messy. A new issue has been made to handle generation of the Python package API (#76).
